tests/automation_framework/src/libs/test_base.py

```diff
@@ -59,7 +57,7 @@ def setup_and_build_request_retrieve(self, input_file):
              'action_obj': action_obj})
         return 0
 
-    def setup_and_build_request_receipt(self, input_file):
+    def setup_and_build_request_create_receipt(self, input_file):
```

Comment:
Renamed the setup_and_build_request_receipt to setup_and_build_request_create_receipt